dali/operators/reader/loader/interleave_mode.h

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Nitpick: update year.

Reply:
Done

Reply:
Feel free to left your copyright if the organization you belong to requires it (or you feel you just want).